code/game/gamemodes/nuclear/pinpointer.dm

```diff
@@ -11,8 +11,12 @@
 	materials = list(MAT_METAL=500)
 	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | ACID_PROOF
 	var/obj/item/disk/nuclear/the_disk = null
-	var/active = 0
+	var/obj/machinery/nuclearbomb/the_bomb = null
+	var/obj/machinery/nuclearbomb/syndicate/the_s_bomb = null // used by syndicate pinpointers.
+	var/active = FALSE
+	var/mode = FALSE // Mode 0 locates disk, mode 1 does something else.
```

Comment:
Youre using `FALSE` for non-booleans (Active can be 0,1,2 in this PR)

Please change

Reply:
Dealt with!